Makefile

```diff
@@ -88,7 +94,19 @@ endif
 	docker run $(NODE_CONTAINER_NAME) calico-felix --version
 	docker run $(NODE_CONTAINER_NAME) libnetwork-plugin -v
 
-	@echo "\nNow push the tag and images. Then create a release on Github and attach the dist/calicoctl binary"
+	@echo "\nNow push the tag and images. Then create a release on Github and \
```

Comment:
I think it'd be cleaner to just use another echo here rather than a line-continuation followed by a `\n`, especially when that's what we do for the lines below:
```
@echo foo
@echo bar
```
Instead of 
```
@echo foo\
\nbar
```